orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/model/Application.groovy

```diff
@@ -29,6 +29,8 @@ class Application {
   String accounts
   String updateTs
   String createTs
+  public Boolean platformHealthOnly;
```

Comment:
Needs the scope modifier because it's accessed from java ... but doesn't need the semi-colons.

These jackson classes are somewhat bogus, I'll make everything public for consistency.